### PR TITLE
adding post about unlocking the blog

### PR DESCRIPTION
**Pull request by julien51**
# Description

I am sending this content by email a lot so I though it would be better as a blog post ;)

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
/blog/protocol-for-membership

File: unlock-protocol.com/blog/unlock-this-blog.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Unlock this blog
+subTitle:
+authorName: Julien Genestoux
+publishDate: July 24, 2019
+description:
+image:
+---
+
+Unlock is a [protocol for memberships](/blog/protocol-for-membership). It helps creators monetize their community (fans, supporters, or even simple users) by helping them deploy a lock and sell access keys to this lock (they can chose their currency, price, the duration of the keys and the number of keys they want to sell...).
```
Comment (akeem) [inline unlock-protocol.com/blog/unlock-this-blog.md]: I think that we may want to rephrase this a bit ` It helps creators monetize their community` as the language implies they are monetizing access to the audience as opposed to the content/creation/access etc

File: unlock-protocol.com/blog/unlock-this-blog.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+We decided to create a lock for members of our community. It's a good way for us to show what's possible with a lock. Since our goal was not to make money, we decided to charge _0 eth_ for the keys.
+
+When integrating the lock on this blog, we decided to reflect the membership status in 3 different things:
+
+1. The lower bar at the bottom of every page. ⬇️ Since I am a member, mine shows this:
+
+![member bar](/static/images/blog/unlock-blog/member-bar.png)
+
+2. On each post, users can view and leave [comments](#comments), but only if they are members themselves.
```
Comment (akeem) [inline unlock-protocol.com/blog/unlock-this-blog.md]:
```suggestion
2. On each post, __members__ can view and leave [comments](#comments).
```

File: unlock-protocol.com/blog/unlock-this-blog.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+3. On some posts, we have decided to lock some content. For example, read this Google Slideshow from a presentation I made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).
```
Comment (akeem) [inline unlock-protocol.com/blog/unlock-this-blog.md]:
```suggestion
3. We've locked some exclusive content. For example, this Google Slideshow from a presentation made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).
```

File: unlock-protocol.com/blog/unlock-this-blog.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+# Bundling
+
+One thing to note is that the lock we use on this blog is in no way limited to this site. We could (and will, stay tuned...) re-use the lock on different sites or applications. As a matter of facts, you could even add *our* lock to *your* site or application, if you wanted to only grant access to people who are part of our community. That approach is how creators can deploy "bundles": a single lock is used on several different websites or applications!
+
+# Several locks
+
+Another important detail is that we could very well add multiple locks to this site. Maybe we'd have one of our most loyal fans (with a limit on the number of keys), and maybe we'd have one for our everyone else, using another currency or one which we would "share" with several other applications to be part of their bundle... etc.
```
Comment (smombartz) [inline unlock-protocol.com/blog/unlock-this-blog.md]: maybe worth clarifying that you only need one key to either of those locks to access the content?

**Comment by smombartz** [comment]
image

**Comment by julien51** [comment]
> In general, I think this is good to go with a few minor changes noted. Reading this one of the things think that we may need to consider the audience a bit. I feel like we are assuming that they know about locks, keys, their relationship in our paradigm, as well as NFTs

So, for context, when I reach out to people, they ask about places where they can "try" unlock. At this point, they know about the general aspects of the protocol, as well a NFT.
For now, I have to send them to our blog and I write a quick explanation about the stuff that is being locked (the bar, the comments, ...). This post is just a longer version of this so that what I write stays in the the "dark". 

I agree we need to write a lot more on the blog (and I think @smombartz has been pushing us to do that!), and maybe it's worth writing a general intro too but this is not the object of that post?

File: unlock-protocol.com/blog/unlock-this-blog.md (hunk 4 above)
Comment (julien51) [reply unlock-protocol.com/blog/unlock-this-blog.md]: Good point! Added

**Comment by julien51** [comment]
All fixed PTAL @akeem ?

File: unlock-protocol.com/blog/unlock-this-blog.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+When integrating the lock on this blog, we decided to reflect the membership status in 3 different things:
+
+1. The lower bar at the bottom of every page. ⬇️ Since I am a member, mine shows this:
+
+![member bar](/static/images/blog/unlock-blog/member-bar.png)
+
+2. On each post, __members__ can view and leave [comments](#comments).
+
+3. We've locked some exclusive content. For example, this Google Slideshow from a presentation made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).
```
[the remark this answers is not in the document]
Comment (julien51) [reply unlock-protocol.com/blog/unlock-this-blog.md]: This would be an extremely ling link :/

**Pull request merged**